code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/load/Unifier.kt

```diff
@@ -0,0 +1,91 @@
+package org.jetbrains.research.code.submissions.clustering.load
+
+import com.intellij.ide.impl.ProjectUtil
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.LocalFileSystem
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiManager
+import com.jetbrains.python.PythonFileType
+import org.jetbrains.research.code.submissions.clustering.Submission
+import org.jetbrains.research.code.submissions.clustering.util.addPyFileToProject
+import org.jetbrains.research.code.submissions.clustering.util.getTmpProjectDir
+import org.jetbrains.research.ml.ast.transformations.Transformation
+import org.jetbrains.research.ml.ast.transformations.anonymization.AnonymizationTransformation
+import org.jetbrains.research.ml.ast.transformations.augmentedAssignment.AugmentedAssignmentTransformation
+import org.jetbrains.research.ml.ast.transformations.commentsRemoval.CommentsRemovalTransformation
+import org.jetbrains.research.ml.ast.transformations.comparisonUnification.ComparisonUnificationTransformation
+import org.jetbrains.research.ml.ast.transformations.constantfolding.ConstantFoldingTransformation
+import org.jetbrains.research.ml.ast.transformations.deadcode.DeadCodeRemovalTransformation
+import org.jetbrains.research.ml.ast.transformations.expressionUnification.ExpressionUnificationTransformation
+import org.jetbrains.research.ml.ast.transformations.ifRedundantLinesRemoval.IfRedundantLinesRemovalTransformation
+import org.jetbrains.research.ml.ast.transformations.multipleOperatorComparison.MultipleOperatorComparisonTransformation
+import org.jetbrains.research.ml.ast.transformations.multipleTargetAssignment.MultipleTargetAssignmentTransformation
+import org.jetbrains.research.ml.ast.transformations.outerNotElimination.OuterNotEliminationTransformation
+import org.jetbrains.research.ml.ast.util.sdk.setSdkToProject
+
+private const val MAX_ITERATIONS: Int = 5
```

Comment:
In Kotlin for clear data types we can commit the type: 
`private const val MAX_ITERATIONS = 5`

Reply:
Also it is not a good idea to use top-level consts